linguatec_lexicon/views.py

```diff
@@ -105,11 +105,12 @@ def near(self, request):
     @action(detail=False)
     def search(self, request):
         query = self.request.query_params.get('q', None)
+        lex = self.request.query_params.get('l', None)
         if query is not None:
             query = query.strip()
-
-        queryset = Word.objects.search(query)
-
+        if lex is not None:
+            lex = lex.strip()
+        queryset = Word.objects.search(query,lex)
```

Comment:
Pay attention to missing space after the comma.

Reply:
Thank you, I will try to pay more attention next time

Reply:
Tools like [flake8](https://flake8.pycqa.org/en/latest/) could help you to detect this kind of issues.
```sh
(env) $ pip install flake8
(env) $ flake8 source_file_to_check.py
```